### DISCUSSION TITLE
OSS & Tea Future

### DISCUSSION THREAD

**Discussion by newgamehd02** [teaxyz/white-paper#116] [Q&A]
In the future, OSS-based projects will issue their own ecosystem coins. Will it be validated on TEA? Is this possible? If it happens, will the TEA ecosystem grow and OSS developers will be able to establish their own ecosystems?

**Comment by thomas-borrel** [comment]
Hi - I don't believe that each OSS-based project issuing their own ecosystem coin is a scalable approach given the complexity of doing it (legal structure, etc...). The goal is for OSS project to have its own DAO without the need to introduce their own token, so they get the benefit of the DAO without the burden of a token creation.

**Reply by newgamehd02** [reply]
Thanks for the descriptive answer.
